### PR TITLE
docs: Add additional documentation for auto generated docs

### PR DESCRIPTION
**Pull request by rmi22186**
# Summary

Adding additional documentation here for auto doc generation.

### CODE REVIEW

File: lib/identity/client_error_codes.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// A class of client error codes to be use with the IdentityApiErrorResponse class
```
Comment (mchuangatmp) [inline lib/identity/client_error_codes.dart:1]: just wanted to double-check, does this need to be `///` like other comments?

Comment (rmi22186) [reply lib/identity/client_error_codes.dart:1]: Yep, good catch!

File: lib/identity/client_error_codes.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (rmi22186) [inline lib/identity/client_error_codes.dart:2]:
```suggestion
/// A class of client error codes to be use with the IdentityApiErrorResponse class

```

File: lib/identity/identity_api_error_response.dart [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import 'package:mparticle_flutter_sdk/identity/client_error_codes.dart';
 
-/// An object returned when an identity request fails for some reason.
+/// A class that returns when an identity request fails for some reason.
 ///
 //  This is passed to the handler for both client and server errors.
```
Comment (rmi22186) [inline lib/identity/identity_api_error_response.dart:5]:
```suggestion
///  This is passed to the handler for both client and server errors.
```

**Review by mchuangatmp — APPROVED** [review]

**Pull request merged**